src/main/kotlin/org/kamiblue/client/module/modules/chat/RemoteCommand.kt

```diff
@@ -18,7 +18,7 @@ internal object RemoteCommand : Module(
     description = "Allow trusted players to send commands",
     category = Category.CHAT
 ) {
-    private val allow = setting("Allow", Allow.FRIENDS)
+    private val allow = setting("Allow", Allow.FRIENDS, description = "Who do you want to be able to see your messages.")
```

Comment:
`Who do you want to be able to see your messages.` really doesn't make sense in this context. The setting is to allow people to send commands to you, not see your messages.